docs/general/emulator-support-and-issues.md

```diff
@@ -378,22 +361,46 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 
 - ❌ _Not supported_.
 
-### Saturn
+### Sega 32X
 
-- ✅ libretro core: **Beetle Saturn**
+- ✅ BizHawk core: **PicoDrive**
+  - Most recommended.
+- ✅ libretro core: **PicoDrive**
+  - Several games are problematic, use BizHawk if an achievement shows as unsupported.
+  - Appears to still have unmapped RAM.
 
 ### Sega CD
 
 - Appears to still have unmapped RAM.
 - ✅ libretro core: **Genesis Plus GX**
 - ✅ libretro core: **PicoDrive**
 
+### Sega Dreamcast
+
+- ✅ libretro core: **Flycast**
+  - Disable threaded rendering to properly use save states.
+
+### Sega Genesis/Mega Drive
+
+- ✅ libretro core: **Genesis Plus GX**
+  - Most recommended.
+- ✅ libretro core: **PicoDrive**
+- ✅ BizHawk core: **Genplus-gx** (Genesis Plus GX)
+
+### Sega Saturn
+
+- ✅ libretro core: **Beetle Saturn**
+
+### Sega Pico
+
+- ❌ _Not supported_.
```

Comment:
This was moved, but should it be removed (along with all the other systems we don't yet support)?

Reply:
I support removal of all these from this page to keep it more concise. Updated in latest.